lizzy_client/cli.py

```diff
@@ -116,19 +133,34 @@ def create(definition: str, image_version: str, keep_stacks: Optional[int],
 
     lizzy = Lizzy(lizzy_url, access_token)
 
+    if not force:  # pragma: no cover
+        # supporting artifact checking would imply copying a large amount of code
+        # from senza, so it should be considered out of scope until senza
+        # and lizzy client are merged
+        warning("Artifact checking is still not supported by lizzy-client.")
```

Comment:
Why not just add a flag to the lizzy API in the BE and allow client to set it? Would leave all the artifact validation etc to senza and client code will be much simpler.